config/recipes/beats/README.asciidoc

```diff
@@ -26,9 +26,9 @@ Deploys Filebeat as a DaemonSet with the autodiscover feature enabled. Logs from
 
 Deploys Filebeat as a DaemonSet with the autodiscover feature disabled. Uses the entire logs directory on the host as the input source. This configuration does not require any RBAC resources as no Kubernetes APIs are used.
 
-==== Metricbeat for Elasticsearch and Kibana Stack Monitoring - `stack_monitoring.yaml`
+==== Elasticsearch and Kibana Stack Monitoring - `stack_monitoring.yaml`
```

Comment:
Update docs in one go?

Reply:
It's a draft for now until the other is merged, then will do that too.